src/components/modules/card/card.styles.ts

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import type { StyledCardImageWrapperProps } from './card.types';
+
+import { styled } from '@/styles';
+
+export const StyledCardWrapper = styled.div`
+  max-width: 656px;
+  width: 100%;
+  background-color: ${({ theme }) => theme.colors.inkBlack};
+  border-radius: 12px;
+  box-shadow:
+    0px 2px 24px 0px rgba(230, 232, 242, 0.05),
+    0 0 0 1px ${({ theme }) => theme.colors.borderGradient};
```

Comment:
Could you add this value `border: 'var(--button-boder)'` to the `colorTheme` and use it here instead of `borderGradient`, please? This border gradient is transparent in light mode, which does not match the design of the border card.